### DISCUSSION TITLE
CLI and API projects/folder creation feature

### DISCUSSION THREAD

**Discussion by manjarres09** [Infisical/infisical#7882] [Feature Request]
### Feature description
Allows users to create projects and folders using the CLI and API. This feature would enable developers to streamline their workflow by automating the project and folder creation process.

### Why would it be useful?
This feature would be highly beneficial for Infisical users, by providing the ability to integrate a step on their pipelines to create projects and folders automatically.

**Comment by jmls** [comment]
I was a little surprised to see that there isn't a full api that the UI would be utilising.  I was investigating infisical as an option to vault but we need an api to provision and maintain orgs, projects, users etc which I can't seem to find any details

Or am I just too old and blind ? :)
